feature/bgp/tests/local_bgp_test/local_bgp_test.go

```diff
@@ -144,7 +162,7 @@ func TestEstablish(t *testing.T) {
 	statePath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
 	nbrPath := statePath.Neighbor(ateAttrs.IPv4)
 	// Remove any existing BGP config
-	gnmi.Delete(t, dut, dutConfPath.Config())
+	bgpClearConfig(t, dut)
 	gnmi.Delete(t, ate, ateConfPath.Config())
```

Comment:
An example of the error generated by this line:
```
Delete(t) on dut2(xx29.den85) at /network-instances/network-instance[name=default]/protocols/protocol[identifier=BGP][name=BGP]: Delete(t) at path origin:"openconfig"  elem:{name:"network-instances"}  elem:{name:"network-instance"  key:{key:"name"  value:"default"}}  elem:{name:"protocols"}  elem:{name:"protocol"  key:{key:"identifier"  value:"BGP"}  key:{key:"name"  value:"BGP"}}: rpc error: code = NotFound desc = error on request {
        prefix:  {
          target:  "xx29.den85"
        }
        delete:  {
          origin:  "openconfig"
          elem:  {
            name:  "network-instances"
          }
          elem:  {
            name:  "network-instance"
            key:  {
              key:  "name"
              value:  "default"
            }
          }
          elem:  {
            name:  "protocols"
          }
          elem:  {
            name:  "protocol"
            key:  {
              key:  "identifier"
              value:  "BGP"
            }
            key:  {
              key:  "name"
              value:  "BGP"
            }
          }
        }
        }: failed to apply: Request could not be completed because leafref at path "/network-instances/network-instance[name='default']/tables/table[protocol='BGP'][address-family='IPV4']/config/protocol" had error "leaf value (openconfig-policy-types:BGP) not present in reference path (../../../../protocols/protocol/config/identifier)"
```

Reply:
Looks like 3c43559 fixes this.